scipy/stats/tests/test_continuous_basic.py

```diff
@@ -63,7 +63,8 @@
                      'gompertz', 'johnsonsb', 'johnsonsu', 'kappa4',
                      'ksone', 'kstwo', 'kstwobign', 'levy_stable', 'mielke', 'ncf',
                      'ncx2', 'powerlognorm', 'powernorm', 'rdist',
-                     'recipinvgauss', 'trapz', 'vonmises', 'vonmises_line']
+                     'recipinvgauss', 'trapz', 'vonmises', 'vonmises_line',
+                     ]
```

Comment:
This looks like a spurious change.  Is it left over from some previous changes that have been undone?
```suggestion
                     'recipinvgauss', 'trapz', 'vonmises', 'vonmises_line']
```

Reply:
That is correct.